cmd/trade.go

```diff
@@ -227,30 +230,31 @@ func init() {
 				}
 			}
 
-			log.Printf("Starting fill tracker with %d handlers\n", fillTracker.NumHandlers())
+			l.Infof("Starting fill tracker with %d handlers\n", fillTracker.NumHandlers())
 			go func() {
 				e := fillTracker.TrackFills()
 				if e != nil {
-					log.Println()
-					log.Printf("problem encountered while running the fill tracker: %s\n", e)
+					l.Info("")
+					l.Info("problem encountered while running the fill tracker:")
+					logger.Fatal(l, e)
 					// we want to delete all the offers and exit here because we don't want the bot to run if fill tracking isn't working
-					deleteAllOffersAndExit(botConfig, client, sdex)
+					deleteAllOffersAndExit(l, botConfig, client, sdex)
 				}
 			}()
 		} else if strategyFillHandlers != nil && len(strategyFillHandlers) > 0 {
-			log.Println()
-			log.Printf("error: strategy has FillHandlers but fill tracking was disabled (set FILL_TRACKER_SLEEP_MILLIS to a non-zero value)\n")
+			l.Info("")
+			l.Info("error: strategy has FillHandlers but fill tracking was disabled (set FILL_TRACKER_SLEEP_MILLIS to a non-zero value)")
```

Comment:
Errorf instead of Info

Reply:
No need for the formatting; using `Error` here